schematic/manifest/generator.py

```diff
@@ -1563,10 +1563,35 @@ def get_manifest(
                                                       output_path = output_path,
                                                       sheet_url = sheet_url,
                                                       empty_manifest_url=empty_manifest_url,
-                                                      dataframe = manifest_df,
+                                                      dataframe = updated_df,
+                                                      out_of_schema_columns = out_of_schema_columns,
                                                       )
             return result
 
+    def _get_end_columns(self, current_schema_headers, existing_manifest_headers, out_of_schema_columns):
```

Comment:
@mialy-defelice this function looks good. Could you also add a unit test to make sure that this function works correctly!